benches/encryption_bench.rs

```diff
@@ -0,0 +1,156 @@
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+use tokio::runtime::Runtime;
+use rustc_hash::FxHashMap;
+use cripta::{
+    app::AppState,
+    config,
+    core::crypto::custodian::Custodian,
+    types::{
+        core::{DecryptedData, DecryptedDataGroup, Identifier},
+        method::EncryptionType,
+    },
+};
+
+const SINGLE_BENCH_ITERATION: u32 = 10;
+const BATCH_BENCH_ITERATION: u32 = 10;
+
+criterion_main!(benches);
+criterion_group!(benches, criterion_data_encryption_decryption, criterion_batch_data_encryption_decryption);
+
+pub fn criterion_data_encryption_decryption(c: &mut Criterion) {
+    let rt = Runtime::new().unwrap();
+    let custodian = Custodian::new(Some(("key".to_string(),"value".to_string())));
+    let config = config::Config::with_config_path(config::Environment::which(), None);
```

Comment:
You can directly use Environment::Dev

Reply:
Sure,cWill update code

Reply:
Updated, please check the code, also create Key using the curl below